### PR TITLE
add in new date to cut release branch

### PR DESCRIPTION
**Pull request by davejrt**
This adds in the param for the calendar event to cut the release branch, as discussed in our sync and [here](https://github.com/sourcegraph/about/pull/3719)

### CODE REVIEW

File: dev/release/release-config.jsonc [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "previousRelease": "3.29.1",
     "upcomingRelease": "3.30.0",
     // Release dates
+    "oneWorkingDayAfterRelease": "19 July 2021 10:00 PST",
```
Comment (daxmc99) [inline dev/release/release-config.jsonc:19]:
```suggestion
    "oneWorkingDayBeforeRelease": "19 July 2021 10:00 PST",
```

**Review by daxmc99 — COMMENTED** [review]
Small typo

File: dev/release/release-config.jsonc (hunk 1 above)
Comment (davejrt) [reply dev/release/release-config.jsonc]: ah, thanks. Fixed now

**Review by daxmc99 — APPROVED** [review]

**Comment by bobheadxi** [comment]
For context, this was intentionally removed in https://github.com/sourcegraph/sourcegraph/pull/17401 as part of the goal to make releases less "eventful" (i.e. avoid instances of rushing things in before branch cut)

**Comment by davejrt** [comment]
@bobheadxi I recall this...but since then we've found dong all of this on the same day to be still too time consuming and risky. For the time being until we get around to improving CI I think this is a better option.

**Pull request merged**